server/index.ts

```diff
@@ -0,0 +1,19 @@
+const path = require("path");
+const express = require('express');
+
+const app = express();
+const PORT = 3000;
```

Comment:
Часто порт нужно указать вне кода, например из переменной окружения.
```suggestion
const PORT = process.env.PORT || 3000;
```

тогда скрипт будет запускаться так 
``` PORT=3000 node server/index.js ```

Reply:
Спасибо, тоже исправил.